src/controllers/profile.js

```diff
@@ -0,0 +1,122 @@
+const User = require('../database/models').User;
+// const model = require('../database/models');
+import model from '../database/models';
+
+const {decodeToken} = require('../utils/decodeToken');
+require('dotenv').config();
+const {validateProfile} = require('../middlewares/profileValidator');
+const _ = require('lodash');
+
+const udpateProfile = async(req,res) => {
+const userId = req.params.userId;
+const  loggedInUser = await decodeToken(req);
+  let user;
+  try{
+    user = await model.User.findByPk(loggedInUser.id);
+    if(!user)
+    return res.status(404).json({success: false, message:"User not found1"});
+    
+  
+    if(user.dataValues.id != userId)
+    return res.status(401).json({success: false, message: "Access  not denied. You can only update your profile"});
+  }
```

Comment:
Consider that we are supporting multiple language, so revisit the way to provided your messages can't be supported for for multiple languages..... work on it and make sure messages are translated in all configured languages.

Reply:
Noted, thanks